### ISSUE TITLE
How to internationalize the grid?

### ISSUE THREAD

**Issue by eestein** [IgniteUI/igniteui-angular#5585]
## Question  
I was reading https://www.infragistics.com/products/ignite-ui-angular/angular/components/localization.html and I'd like to confirm, before taking this route, this is the way to do it.
Should I really create a variable with all resources of the grid, manually translated, per language? Aren't there no default resources for "common" languages like German, French, Italian, etc? Only Japanese and Korean by downloading the i18n package?

Is this correct or am I missing something?

 * igniteui-angular version: 8.1.0
 * browser: Firefox

**Comment by zdrawku** [comment]
Yes, your assumption is correct, currently, we only have [Spanish, Korean and Japanese](https://github.com/IgniteUI/igniteui-angular-i18n/tree/master/src/i18n) translations, and in order to use other languages, you may consider adding them first in the i18n repository.

**Comment by kdinev** [comment]
You can also reuse the object structure for the i18n resources, and repopulate them with a translation in a language that is not currently available in what we've provided.

You can find guidance here: https://www.infragistics.com/products/ignite-ui-angular/angular/components/localization.html#localize-particular-strings-for-all-components

**Comment by eestein** [comment]
@kdinev Thanks, I read all of that, I was just "amazed" to see that "common" languages like the ones I've mentioned are not already available and preferred to confirm before doing all of this work.

**Comment by kdinev** [comment]
@eestein The languages already available are mostly based on our ability to translate the product internally. The only language that we can also translate in, but that is not yet provided is Bulgarian. If there's demand however for other languages like German, French, Italian, etc, then we will definitely try to provide them out of the box! Also, we accept contributions to the i18n repo and we will be more than happy to include any language provided by an outside contributor.
